index.html

```diff
@@ -8,206 +9,244 @@
-                <li><a href="#"><img src="./ImageP1/Linkedin.png" alt="Linkedin-icon"></a></li>
-                <li><a href="#"><img src="./ImageP1/medium.png" alt="Medium-icon"></a></li>
-                <li><a href="https://github.com/Bhekisile/Bee_Portfolio"><img src="./ImageP1/github@.png" alt="Github-icon"></a></li>
-                <li><a href="#"><img src="./ImageP1/angellist@.png" alt="Angellist-icon"></a></li>
-            </ul>   
-        </header>
+
+            <header class="myheader">
+                <h1> <span class="blue">Hey there. <br>
+                        I’m Alex. <br></span>
+                    <span class="green"> I’m a software <br>
+                        developer</span>
+                </h1>
+            </header>
+            <article>
+                <p>I can help you build a product , feature or website Look through some of my work and experience! If
+                    you like what you see and have a project you need coded, <br>
+                    don’t hesitate to contact me.</p>
+            </article>
+            <div class="flex-icons">
+                <ul>
+                    <li><a href="#"><img src="./ImageP1/twitter@.png" alt="Twitter-icon"></a></li>
+                    <li><a href="#"><img src="./ImageP1/Linkedin.png" alt="Linkedin-icon"></a></li>
+                    <li><a href="#"><img src="./ImageP1/medium.png" alt="Medium-icon"></a></li>
+                    <li><a href="https://github.com/Bhekisile/Bee_Portfolio"><img src="./ImageP1/github@.png"
+                                alt="Github-icon"></a></li>
+                    <li><a href="#"><img src="./ImageP1/angellist@.png" alt="Angellist-icon"></a></li>
+                </ul>
+            </div>
+
         </section>
         <div class="chevron">
             <img src="./ImageP1/chevron1.png" alt="Chevron">
         </div>
-    
-    <section class="works">
-        <header>
-            <h2 class="work">My Recent <br> Works</h2>
-        </header>
-        <div class="indicator">
-            <img src="./ImageP1/Indicator.png" alt="Indicator">
-        </div>
-        <main class="cards">
-            <div class="card">
-                <div class="card-img">
-                </div>
-                <section>
-                    <header>
-                        <h3>Multi-Post Stories <br> Gain+Glory</h3>
-                    </header>
-                    <div class="languages">
-                        <ul>
-                            <li class="ruby">Ruby on Rails</li>
-                            <li class="css">CSS</li>
-                            <li class="js">JavaScript</li>
-                            <li class="html">HTML</li>
-                        </ul>
-                    </div>
-                    <button type="button" class="btn">See project</button>
-                </section>
+
+        <section class="works">
+            <header>
```

Comment:
- [ ] [OPTIONAL] I found out that all the `section` tags are wrapped around the `header` tag which is a wrong implementation of the HTML semantic tag. The `header` tag should wrap around only the `nav` tag. This is a [link](https://developer.mozilla.org/en-US/docs/Glossary/Semantics) for HTML semantic tags. 😄